migrations/postgres/20250415100644_device_codes.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE device_codes
```

Comment:
let's clearly indicate this is related to oauth in our names

Reply:
who knows what kind of IoT device modeling we will have tomorrow

Reply:
Yes, you're right -- the previous name was too generic

Fixed